src/tests/functional/plugin/cpu/test_utils/cpu_test_utils.hpp

```diff
@@ -157,6 +157,9 @@ class CPUTestsBase {
     std::vector<cpu_memory_format_t> inFmts, outFmts;
     std::vector<std::string> priority;
     std::string selectedType;
+    std::vector<CPUSpecificParams> cpuParamsVec;
+    std::vector<double> relThresholds;
```

Comment:
Yes, you're right. I moved this vector to `ConvolutionLayerCPUTest` class for now. 
But in the future, it would be nice to inherit `ConvolutionLayerCPUTest` and `GroupConvolutionLayerCPUTest`classes from a generic class and define `relThresholds` there if needed.